lib/youtube-dl.js

```diff
@@ -316,6 +320,12 @@ ytdl.getInfo = function getInfo(url, args, options, callback) {
     if (err) return callback(err);
     let info;
 
+    // If using the "-g" or "--get-url" flag youtube-dl will return just a string (the URL to the video) which messes up the parsing
+    // This fixes this behaviour
+    if (args && (args.indexOf("-g") > -1 || args.indexOf("--get-url") > -1))
```

Comment:
The same thing here, this kind of checks are around the code, could be nice start extracting it to make the code more readable:

```js
const has = (collection, value) => collection && collectionindexOf(value)  > -1
```

Reply:
Done in 5ec1f87

Left [this](https://github.com/przemyslawpluta/node-youtube-dl/blob/113ed2a1d61e434f35d7fa173770043c80704ddd/lib/youtube-dl.js#L313) and [this](https://github.com/przemyslawpluta/node-youtube-dl/blob/113ed2a1d61e434f35d7fa173770043c80704ddd/lib/youtube-dl.js#L376) untouched as they're checking for a special index didn't want to break anything that's beyond the scope of this PR already.